aws-operator-role/iam-policy.json

```diff
@@ -12,15 +12,15 @@
                 "iam:AttachRolePolicy",
                 "iam:CreateInstanceProfile",
                 "iam:CreateOpenIDConnectProvider",
-		"iam:DeleteOpenIDConnectProvider",
+		        "iam:DeleteOpenIDConnectProvider",
                 "iam:ListOpenIDConnectProviderTags",
                 "iam:TagOpenIDConnectProvider",
                 "iam:UntagOpenIDConnectProvider",
                 "iam:ListOpenIDConnectProviders",
                 "iam:GetOpenIDConnectProvider",
-		"iam:UpdateOpenIDConnectProviderThumbprint",
-		"iam:RemoveClientIDFromOpenIDConnectProvider",
-		"iam:AddClientIDToOpenIDConnectProvider",
+		        "iam:UpdateOpenIDConnectProviderThumbprint",
```

Comment:
seems wrong indentation

Reply:
yeah, just fixed, there were some tabs in there